src/test/java/unit/BqDatasetReferenced.java

```diff
@@ -309,60 +309,63 @@ void updateIndividualProperties() throws IOException {
     // update just the name
     // `terra resources update bq-dataset --name=$name --new-name=$newName`
     String newName = "updateIndividualProperties_NEW";
-    UFBqDataset updateDataset =
+    UFBqDataset updatedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class,
             "resource",
             "update",
             "bq-dataset",
             "--name=" + name,
             "--new-name=" + newName);
-    assertEquals(newName, updateDataset.name);
-    assertEquals(description, updateDataset.description);
+    assertEquals(newName, updatedDataset.name);
+    assertEquals(description, updatedDataset.description);
 
     // `terra resources describe --name=$newName`
-    UFBqDataset describeDataset =
+    UFBqDataset describedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class, "resource", "describe", "--name=" + newName);
-    assertEquals(description, describeDataset.description);
+    assertEquals(description, describedDataset.description);
 
-    // update just the description
-    // `terra resources update bq-dataset --name=$newName --description=$newDescription`
+    // update description and cloning instructions
+    // `terra resources update bq-dataset --name=$newName --description=$newDescription
+    // --new-cloning=COPY_NOTHING`
```

Comment:
Here you have COPY_NOTHING, but on 341 you have NOTHING. For all tests, please make sure your comments are in sync